src/test/java/de/retest/ui/descriptors/ElementTest.java

```diff
@@ -18,7 +18,8 @@
 	public void toString_returns_UniqueCompIdentAttributes_toString() throws Exception {
 		final IdentifyingAttributes compIdentAttributes =
 				IdentifyingAttributes.create( fromString( "Window/path/Component" ), java.awt.Component.class );
-		assertThat( new Element( compIdentAttributes, null ).toString() ).isEqualTo( compIdentAttributes.toString() );
+		assertThat( new Element( "", compIdentAttributes, null ).toString() )
```

Comment:
Isn't this a flaw that we allow IDs to be empty? Should be like a variable, at least one alphanumeric char (IMHO).

Reply:
Good catch.